wiki/en/en-Adminstration.md

```diff
@@ -5,4 +5,172 @@ lang: "en"
+
+We use the following suffixes for pre-releases:
+  - `dev` to denote custom builds from git. Example: `3.7.0dev-123456` is an unofficial release built from code on top of the final 3.7.0 release.
+  - `devNightly` to denote deliberate snapshots from the official jamulussoftware/jamulus git. Example: `3.7.0devNightly1-123456` is an official pre-release built from code on top of the final 3.7.0 release. There may be zero, one or multiple nightly builds during a release cycle.
+  - `beta` to denote test builds for new releases. Example: `3.8.0beta1` is the first official test build before 3.8.0 final is tagged. Betas are still subject to change.
+  - `rc` to denote release candidates. Example: `3.8.0rc1`is the first release candidate before 3.8.0 final is tagged. Release candidates must not receive any code changes. If they do (e.g. due to critical bugfixes), another rc has to be tagged.
+
+The git tag is prefixed by `r` (for release) and all dots are replaced by underscores.
+Example: The git tag `r3_7_0beta1` refers to the user-visible version `3.7.0beta1`.
+
+### Forks
+Forks of Jamulus must not use the same versioning scheme if they publish their own releases.
+Having a jamulussoftware/jamulus-3.7.0 release and a example.org/jamulus-3.7.0 release would be confusing to users.
+Instead, forks must distinguish their version, e.g. by adding an additional suffix.
+Example: 3.7.0-example.org-beta1
+
+## Creating a (pre)release
+  - Feature freeze, Code freeze
+  - It's ok if translation PRs are not ready
+  - Use issue template(s)?
+  - Release shepherd?
+  - Sign release tag?
+  - Release announcements
+  - Close the milestone
+
+## Changelog
+  - Biggest changes first
+  - Refactorings, internal-only or build-related changes last
+  - Full sentences on a single line, followed by issue and/or PR references in parantheses. Sentence should make the scope (e.g. Client, Server, UI, ...) clear as early as possible.
+  - Optionally, a longer description in a subsequent line.
+  - `(contributed by @<Github handle>)` in a subsequent line.
+
+## Translations
+TODO: link to translation-specific documentation
+
+# Project organization
+
+## Long-term goals
+TODO: Link to / integrate @gilgongo's document?
+
+## Project board
+https://github.com/jamulussoftware/jamulus/projects/10?card_filter_query=milestone%3A%22release+3.7.0%22,
+https://github.com/jamulussoftware/jamulus/discussions/929#discussioncomment-420872
+
+## Milestones
+
+## Issues and Pull Requests for jamulussoftware/jamulus
+<!-- TODO: Should we extend this to work for jamuluswbesite as well? If so, how? -->
+We aim to give early feedback and try to keep an overview by using the Project board.
+
+### Triage
+When a new Issue or Pull Request comes in, do the following:
+1. Add it to the Tracking Project board.
+1. Check for duplicates.
+1. Decide whether to accept this Issue/PR:
+   1. Reject if it is a feature which goes against Jamulus' core principles. <!-- TODO: Link CONTRIBUTE.md -->
```

Comment:
Rather than a list of if/thens, it might be easier to just say that "GH Issues should be a list of agreed things that can be done without having to ask for more info or clarification. This means that unless an Issue is a clearly specced statement of work that has a link to a prior discussion, or is a clear (as in reproducible/easily observable) bug, then it gets moved to discussion so it can later be submitted as such an Issue or a PR."

Reply:
For the the "Triage" section, I suggest:

Issues:

These should be a list of agreed things that can be done without having to ask for more info or clarification.

Issues that have not been discussed, or are vague or non-reproducible bug reports, should be moved to Discussions with a short note linking to these instructions if necessary (the latter can be put in the "Bug Investigation" category). Remind the poster they can open a new Issue or PR once the Discussion has reached agreement on what to implement.

PRs:

Similarly to Issues, these should have had some prior discussion, with their authors aware of the contributing guidelines. PRs without a reference to such discussion should be given a short note linking to these instructions and/or contributing guidelines if necessary. Remind the poster they can resume the PR once a GH Discussion about it has reached agreement.

Issues and PRs:

The submitter of an Issue or a PR is responsible for its care and feeding, answer all questions directed at them, and make agreed amends if needed. 

So as to reduce effort for others in understanding a PR or an Issue, authors are strongly encouraged to update their initial posts/PR descriptions or title to reflect the current state of play, amends, enhancements, outstanding issues etc. Admins reserve the right to do this for them if need be.